### PR TITLE
create edit delete book author category

### PR DESCRIPTION
**Pull request by phornsopheak**

### CODE REVIEW

**Comment by RathanakSreang** [comment]
Ok

**Comment by HorNorin** [comment]
ok

File: app/controllers/admins/authors_controller.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      render "new"
+    end
+  end
+
+  def edit
+    @authors =  Author.paginate page: params[:page] 
+  end
+
+  def update
+    if @author.update_attributes(author_params)
```
Comment (chuanhtuan) [inline app/controllers/admins/authors_controller.rb]: ()

File: app/controllers/admins/books_controller.rb [hunk 2]
```diff
@@ -17,8 +22,37 @@ def create
     end
   end
 
+  def edit
+    @categories = Category.all
+    @authors = Author.all
+  end
+
+  def update
+    if @book.update_attributes(book_params)
```
Comment (chuanhtuan) [inline app/controllers/admins/books_controller.rb]: ()

File: app/controllers/admins/categories_controller.rb [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      render "new"
+    end
+  end
+
+  def edit
+    @categories =  Category.paginate page: params[:page] 
+  end
+
+  def update
+    if @category.update_attributes(category_params)
```
Comment (chuanhtuan) [inline app/controllers/admins/categories_controller.rb]: ()

**Comment by samnangcattor** [comment]
ok

**Pull request merged**